Extra_tasks/Extra_task_7/Extra_task_7.1/script/script.js

```diff
@@ -0,0 +1,4 @@
+createList('Заголовок', tree);
+
+const ul = document.querySelector('ul');
+ul.addEventListener('click', hideElem);
```

Comment:
I prefere write callBack Here. For example:
ul.addEventListener('click', ( (event) => {
  const elem = event.target;
  const sibling = elem.nextSibling;

  if (sibling?.tagName === 'UL') {
    sibling.classList.toggle('hide');
  }
});